spec/ATOMICSETTLEMENT.mediawiki

```diff
@@ -0,0 +1,161 @@
+==Atomic Settlement==
+
+In order to facilitate trustless, non-custodial exchange, the DEX leverages an
+atomic swap process that enables all parties to maintain full control over their
+assets at every step.
+The DEX primarily acts as a matchmaker, connecting two orders and relaying
+messages between the two clients.
+Additionally, the DEX performs some trade monitoring to ensure that parties are
+acting in good faith.
+Clients who fail to act on their trades appropriately will be issued a penalty
+in accordance with the [[#community-conduct|rules of community conduct]].
+
+The following cases are meant to provide a high-level overview of the trade
+process, and to illustrate how to handle a perfect match, a partial fill, and
+multiple matches within an epoch. For the sake of illustration, the following
+simplifications are made.
+
+<img src="images/ab-profiles.png" align="right">
+# The lot size for the market is 1 DCR.
+# All backing funds for the orders are in the form of a single coin.
+# There are no fees.
+
+The examples involve a participant named Alice, who holds some DCR and would
+like to exchange it for some BTC.
+The market is a DCR/BTC market, where DCR is the $B!H(Bbase asset$B!I(B and BTC is the
+$B!H(Bquote asset$B!I(B.
```

Comment:
Hwat? 

These characters are all over this file.

Reply:
They are double quotations and apostrophes.  They look fine in my editor... hmm...